aiplatform/src/test/java/aiplatform/imagen/EditImageInpaintingInsertMaskSampleTest.java

```diff
@@ -44,7 +44,6 @@ private static void requireEnvVar(String varName) {
 
   @BeforeClass
   public static void checkRequirements() {
-    requireEnvVar("GOOGLE_APPLICATION_CREDENTIALS");
```

Comment:
Do we not need this env variable set for the tests?

Reply:
Added back in